ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1159,6 +1159,56 @@ def test_tensorflow_realdiv(
         y=x[1],
     )
 
+    
+# tile
+@st.composite
+def _multiple_shape_helper(draw):
+    input_dtype, input_array, input_shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["int32", "int64"],
+            min_value=0,
+            max_value=10,
```

Comment:
Also, we don't need `min_value` and `max_value` here, only at the multiples. The array that is being repeated can contain any value.

Reply:
I ran into another error [like this one](https://github.com/unifyai/ivy/pull/7812#discussion_r1083902855), and assumed that it was because of memory.

I understand now that this inputs the values to the input array, and has no effect on how many times it will be repeated

Reply:
should I limit the value of input array dimensions so it does not use too much memory?